lib/config/presets/internal/default.ts

```diff
@@ -456,6 +459,18 @@ export const presets: Record<string, Preset> = {
       'Apply labels <code>{{arg0}}</code> and <code>{{arg1}}</code> to PRs',
     labels: ['{{arg0}}', '{{arg1}}'],
   },
+  labelUpdateTypes: {
+    description:
+      'Apply label per update type, prefixed by <code>{{arg0}}</code> and suffixed by <code>{{arg1}}</code> to PRs',
+    packageRules: [].concat(
+      options
+        .find((it) => it.name === 'matchUpdateTypes')
+        .allowedValues.map((t) => ({
+          matchUpdateTypes: [t],
+          labels: ['{{arg0}}' + t + '{{arg1}}'],
+        }))
+    ),
```

Comment:
```suggestion
    packageRules:
      options
        .find((it) => it.name === 'matchUpdateTypes')
        .allowedValues.map((t) => ({
          matchUpdateTypes: [t],
          labels: ['{{arg0}}' + t + '{{arg1}}'],
        })),
```

The `concat` isn't nesessary, as `map` is already creating a new array.

Reply:
Let me try that again: I did it that way first, but couldn't get it to compile, and this was a quick work around.

Reply:
Addressed it with an `as PackageRule`.